src/components/drag-and-drop-list/drag-and-drop-list.css

```diff
@@ -0,0 +1,39 @@
+.dragAndDropList {
```

Comment:
In CSS files, it is a convention to use kebab case
```suggestion
.drag-and-drop-list {
```